src/mesh/voxel_cloud.rs

```diff
@@ -3220,108 +3156,6 @@ mod tests {
         assert!(analysis::are_similar(&voxel_mesh, &voxel_mesh_synced));
     }
 
-    #[test]
-    fn test_scalar_field_boolean_intersection_all_volume() {
```

Comment:
Would be great if we could bring these tests back. Of course the correct scalar fields will look a bit differently, because the boolean functions don't resize anything.